conans/client/loader.py

```diff
@@ -228,7 +228,7 @@ def _parse_file(conan_file_path):
     try:
         module_id = str(uuid.uuid1())
         current_dir = os.path.dirname(conan_file_path)
-        sys.path.append(current_dir)
+        sys.path.insert(0, current_dir)
```

Comment:
It won't ever happen, but... I would put these three lines outside the `try` block to be sure that the `finally` block removes the inserted path (otherwise, if one of these lines raises, we are removing something else from the path list).

Reply:
Agree, fixed.